app/tests/e2e/test_revocation.py

```diff
@@ -0,0 +1,249 @@
+import pytest
+from assertpy import assert_that
+from fastapi import HTTPException
+
+from app.routes.issuer import router
+from shared import RichAsyncClient
+
+CREDENTIALS_BASE_PATH = router.prefix
+
+
+@pytest.mark.anyio
+async def test_clear_pending_revokes(
+    faber_client: RichAsyncClient, issue_revocable_credentials_to_alice_and_revoke: list
+):
+
+    response = (
+        await faber_client.get(
+            f"{CREDENTIALS_BASE_PATH}/revocation/record"
+            + "?credential_exchange_id="
+            + issue_revocable_credentials_to_alice_and_revoke[0]["credential_id"][3:]
```

Comment:
a bit clearer to assign this `issue_revocable_credentials_to_alice_and_revoke[0]["credential_id"][3:]` to a variable: faber_cred_ex_id?